### ISSUE TITLE
NameError: name 'basestring' is not defined (using beets 1.4.7)

### ISSUE THREAD

**Issue by kimpenhaus** [ageorge/beets-bandcamp#8]
Hi guys,

after upgrading to beets 1.4.7 recently it looks like bandcamp plugin is broken.
While trying to import an album with tagging from bandcamp I am receving following error:

`NameError: name 'basestring' is not defined`

StackTrace is as follows:

```
Traceback (most recent call last):
  File "/usr/local/bin/beet", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/site-packages/beets/ui/__init__.py", line 1256, in main
    _raw_main(args)
  File "/usr/local/lib/python3.6/site-packages/beets/ui/__init__.py", line 1243, in _raw_main
    subcommand.func(lib, suboptions, subargs)
  File "/usr/local/lib/python3.6/site-packages/beets/ui/commands.py", line 943, in import_func
    import_files(lib, paths, query)
  File "/usr/local/lib/python3.6/site-packages/beets/ui/commands.py", line 913, in import_files
    session.run()
  File "/usr/local/lib/python3.6/site-packages/beets/importer.py", line 329, in run
    pl.run_parallel(QUEUE_SIZE)
  File "/usr/local/lib/python3.6/site-packages/beets/util/pipeline.py", line 445, in run_parallel
    six.reraise(exc_info[0], exc_info[1], exc_info[2])
  File "/usr/local/lib/python3.6/site-packages/six.py", line 693, in reraise
    raise value
  File "/usr/local/lib/python3.6/site-packages/beets/util/pipeline.py", line 312, in run
    out = self.coro.send(msg)
  File "/usr/local/lib/python3.6/site-packages/beets/util/pipeline.py", line 194, in coro
    func(*(args + (task,)))
  File "/usr/local/lib/python3.6/site-packages/beets/importer.py", line 1511, in plugin_stage
    func(session, task)
  File "/usr/local/lib/python3.6/site-packages/beets/plugins.py", line 140, in wrapper
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/beetsplug/fetchart.py", line 807, in fetch_art
    candidate = self.art_for_album(task.album, task.paths, local)
  File "/usr/local/lib/python3.6/site-packages/beetsplug/fetchart.py", line 873, in art_for_album
    for candidate in source.get(album, self, paths):
  File "/usr/local/lib/python3.6/site-packages/beetsplug/bandcamp.py", line 296, in get
    if isinstance(album.mb_albumid, basestring) and u'bandcamp' in album.mb_albumid:
NameError: name 'basestring' is not defined
```

`beet version` gives me:

```
beets version 1.4.7
Python version 3.6.5
plugins: bandcamp, chroma, discogs, duplicates, edit, embedart, fetchart, lastgenre, missing, scrub, zero
```

In my config I have bandcamp art enabled:

```
bandcamp:
    art: true
``` 

And I am using the latest version of `beets-bandcamp 0.1.2`
If there is anything else I need to provide I will gladly do.

Disabling art in config gives me the ability to import - so as a workaround one can download the art and put into the import folder.

Thanx!

**Comment by unrblt** [comment]
Thanks for the detailed report. I've just released the `0.1.3` version that works with python 3.

**Comment by kimpenhaus** [comment]
Updated - checked - works like a charm :-)

Thanks!
